.env

```diff
@@ -25,6 +25,8 @@ HASH_REDIS_PORT=6379
 HASH_SEARCH_LOADER_PORT=3838
 HASH_SEARCH_QUEUE_NAME=search
 
+LOG_LEVEL=info
+
```

Comment:
@benwerner01 I moved `LOG_LEVEL` a bit further up. I did not notice it was under _Aliases for dockerized external services_ when reviewing #322 


<img width="1172" alt="Screenshot 2022-02-17 at 17 02 14" src="https://user-images.githubusercontent.com/608862/154532814-285d0769-7536-46c8-aeec-b8b9bbdcce91.png">

Reply:
My bad, thank you 🙏